src/bp/ui-studio/src/web/components/Content/CreateOrEditModal.tsx

```diff
@@ -1,8 +1,9 @@
+import { Dialog } from '@blueprintjs/core'
 import { lang } from 'botpress/shared'
 import classnames from 'classnames'
 import _ from 'lodash'
 import React from 'react'
-import { Button, Modal } from 'react-bootstrap'
+import { Button } from 'react-bootstrap'
```

Comment:
Bootstrap 😭!   Is this button still used ?

Reply:
I believe it is, I didn't want to refactor that as it was just a fix to the modal and it will most likely all change in the near future anyway